testing/src/cypress/custom-command/common/authorization.ts

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import { stringify } from "querystring";
+
+export default class Authorization {
+  CLIENT_ID = {
```

Comment:
This can be created as the below enum also.
```ts
export enum ClientIdType {
  Student = "student",
  Institution = "institution",
  AEST = "aest",
  SupportingUsers = "supportingUsers",
}
```

Reply:
updated